website/source/docs/auth/token.html.md

```diff
@@ -286,8 +286,10 @@ of the header should be "X-Vault-Token" and the value should be the token.
       <li>
         <span class="param">increment</span>
         <span class="param-flags">optional</span>
-            An optional requested lease increment can be provided. This
-            increment may be ignored.
+            An optional requested lease increment (measured in seconds) can be provided.
+            If changing the token's TTL to current time + `increment` seconds would
```

Comment:
The problem with specifying this here is that there may be other reasons that the increment is ignored. So this may lead people to wonder why their increment is being ignored when it's not going to exceed the max lease TTL.